pkgs/applications/networking/instant-messengers/signald/default.nix

```diff
@@ -18,6 +18,23 @@ let
     sha256 = "sha256-504je6hKciUGelVCGZjxGjHi1qZQaovagXD5PBQP+mM=";
   };
 
+  jre' = jre_minimal.override {
+    jdk = jdk17_headless;
+    modules = [
+      "java.base"
```

Comment:
How did you get the list of these modules? Couldn't it break at runtime if you miss one? Also in the case it's based on some source you should probably add a comment how you got them.

Reply:
I did it on my own by dogfooding this. However, you're right, maintaining this on our own is somewhat dangerous, especially since we'll need someone to thoroughly test it on each package change.

I guess I'll file a PR against upstream to make sure that we can just use whatever they're using.